server/project/project.go

```diff
@@ -40,7 +45,7 @@ func (s *Server) List(ctx context.Context, q *ProjectQuery) (*v1alpha1.AppProjec
 		newItems := make([]v1alpha1.AppProject, 0)
 		for i := range list.Items {
 			project := list.Items[i]
-			if s.enf.EnforceClaims(ctx.Value("claims"), "projects", "get", fmt.Sprintf("*/%s", project.Name)) {
+			if s.enf.EnforceClaims(ctx.Value("claims"), "projects", "get", fmt.Sprintf("%s", project.Name)) {
```

Comment:
Style nit: get rid of fmt.Sprintf() and just use project.Name

Reply:
fixed.